test/integration/org/org_test.go

```diff
@@ -303,6 +303,33 @@ func TestOrg(t *testing.T) {
 
 			}
 
+			// Log Sink billing
+			billingAccount := org.GetStringOutput("billing_account")
+			billingLBKSinkName := org.GetStringOutput("billing_sink_names")[0]
+			billingPUBSinkName := org.GetStringOutput("billing_sink_names")[1]
+			billingSTOSinkName := org.GetStringOutput("billing_sink_names")[2]
```

Comment:
A possible improvement that I see here would be to change these outputs to return a `map` instead of a `list`, so we can access the values like `org.GetStringOutput("billing_sink_names")["lbk"]` instead of the index `[0]`.

If we do that, this test would be less fragile and if for some reason the order of this output is changed, the test will continue to work as expected.

It is just an improvement suggestion, not a blocker. Feel free to evaluate if it is worthy :)

Reply:
+1